Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -522,46 +571,23 @@ section CommMonoid
 
 variable {R : Type*} [CommMonoid R] {S : Submonoid R} [OreSet S]
 
+@[to_additive]
 theorem oreDiv_mul_oreDiv_comm {r₁ r₂ : R} {s₁ s₂ : S} :
     r₁ /ₒ s₁ * (r₂ /ₒ s₂) = r₁ * r₂ /ₒ (s₁ * s₂) := by
   rw [oreDiv_mul_char r₁ r₂ s₁ s₂ r₁ s₂ (by simp [mul_comm]), mul_comm s₂]
 #align ore_localization.ore_div_mul_ore_div_comm OreLocalization.oreDiv_mul_oreDiv_comm
 
+@[to_additive]
 instance : CommMonoid R[S⁻¹] :=
   { OreLocalization.instMonoidOreLocalization with
     mul_comm := fun x y => by
       induction' x using OreLocalization.ind with r₁ s₁
       induction' y using OreLocalization.ind with r₂ s₂
       rw [oreDiv_mul_oreDiv_comm, oreDiv_mul_oreDiv_comm, mul_comm r₁, mul_comm s₁] }
 
-variable (R S)
-
-/-- The morphism `numeratorHom` is a monoid localization map in the case of commutative `R`. -/
-protected def localizationMap : S.LocalizationMap R[S⁻¹] where
-  toFun := numeratorHom
-  map_one' := rfl
-  map_mul' r₁ r₂ := by simp
-  map_units' := numerator_isUnit
-  surj' z := by
-    induction' z using OreLocalization.ind with r s
-    use (r, s); dsimp
-    rw [numeratorHom_apply, numeratorHom_apply, OreLocalization.expand' r 1 s]
-    simp only [mul_div_one, mul_one, Submonoid.smul_def, smul_eq_mul, mul_comm r]
-  exists_of_eq r₁ r₂ := by
-    dsimp
-    intro h
-    rw [numeratorHom_apply, numeratorHom_apply, oreDiv_eq_iff] at h
-    rcases h with ⟨u, v, h₁, h₂⟩
-    dsimp at h₂
-    rw [mul_one, mul_one] at h₂
-    subst h₂
-    exact ⟨u, h₁.symm⟩
-#align ore_localization.localization_map OreLocalization.localizationMap
-
-/-- If `R` is commutative, Ore localization and monoid localization are isomorphic. -/
-protected noncomputable def equivMonoidLocalization : Localization S ≃* R[S⁻¹] :=
-  Localization.mulEquivOfQuotient (OreLocalization.localizationMap R S)
-#align ore_localization.equiv_monoid_localization OreLocalization.equivMonoidLocalization
+-- The comparison with `Localization` is removed as we(@erdOne) plan to replace that.
```

Comment:
But are you replacing it somewhere else in this PR?

Reply:
No but this will become rfl in the next PR.

Reply:
But then I'd keep it here now and remove it in the next PR, no? Or am I missing something? Are you suggesting that we temporarily "deprive" mathlib of this lemma?

Reply:
I would hope that I won't have to touch this file again so I moved these into `GroupTheory/MonoidLocalization.lean` for now.